climada/entity/impact_funcs/base.py

```diff
@@ -127,4 +127,65 @@ def check(self):
                            'origin of the intensity scale. In impact.calc '
                            'the impact is always null at intensity = 0.')
 
+    def set_step_ImpF(self, threshold, inten_min, inten_max, if_id=1):
```

Comment:
the naming is not consistent with the lynting conventions. change to 
```suggestion
    def set_step_impf(self, threshold, inten_min, inten_max, if_id=1):
```
 ?

Reply:
why not. I don't think we're very consistent with this throughout climada and quite many use ImpF as abbreviation. but surely nice to be in line with lynting conventions. See 67385c69